shape-calculator/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Take a bit of time to include either deploy information or screenshots in your Readme and pull request.